packages/react-components/react-tabster/src/hooks/useKeyboardNavAttribute.ts

```diff
@@ -6,7 +6,9 @@ import type { KeyborgCallback } from 'keyborg/dist/Keyborg';
 import type { RefObject } from 'react';
 
 /**
- * instantiates keyborg and add attribute to ensure focus indicator synced to keyborg logic
+ * Instantiates [keyborg](https://github.com/microsoft/keyborg) and adds `data-keyboard-nav`
+ * attribute to a referenced element to ensure keyboard navigation awareness
+ * synced to keyborg logic without having to cause a re-render on react tree.
  */
 export function useKeyboardNavAttribute<E extends HTMLElement>() {
```

Comment:
I'm getting the feeling that the name of this hook is not reflecting enough what it does, since it's actually returning a reference and then modifying attributes directly to the DOM element. WDYT @ling1726? `useRefToKeyboardNavAttribute`  😅 ?

Reply:
well we can't change it now anyway :)